WBooks/Rent.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Rent.swift
+//  WBooks
+//
+//  Created by Juan Silva on 15/06/2021.
+//
+
+import Foundation
+
+struct Rent: Codable {
+    let book_id: Int
+    let user_id: Int
+    let id: Int
+    let from: String
+    let to: String
+}
```

Comment:
In this case, it would be better to name the first properties as `bookId` and `userId`, and then use the coding keys. 

```Swift
struct Rent: Codable {
    ...

    enum CodingKeys: String, CodingKey {
        case bookId = "book_id"
        case userId = "userId"
        case id
        case from
        case to
        case location
    }

}
```

This is just to follow the naming conventions (camelCase instead of snake_case for properties) 😉